app/views/overview/_list-row-content.html

```diff
@@ -39,7 +39,7 @@
   <div ng-if="row.apiObject.kind === 'DeploymentConfig'">
     <span class="mar-right-sm">
       <span class="hidden-xs">
-        {{row.apiObject.spec.strategy.type}} deployment
+        {{row.apiObject.spec.strategy.type}} deployment is
```

Comment:
It should:

is new
is pending
is running
is complete
is failed

https://github.com/spadgett/origin/blob/5dce9b90d3eeab4a3c7a709c604c2a9ec2f7199f/pkg/deploy/api/types.go#L107-L120